testkit/src/main/scala/com/velocidi/apso/elasticsearch/ElasticsearchTestKit.scala

```diff
@@ -40,7 +40,6 @@ trait NoSpecElasticsearchTestKit {
       // another es-cluster-runner instance could already be using that same port
       // By adding this Random we avoid binding to ports already in use
       .baseHttpPort(9200 + Random.nextInt(50))
-      .baseTransportPort(9300 + Random.nextInt(50))
```

Comment:
Do you know the reason for this being removed?

Reply:
Not sure. I couldn't find the reason for this change. 

Commit that remove this setting: https://github.com/codelibs/elasticsearch-cluster-runner/commit/51d2774618e8fbc89ef41d750c25ff32bd43a3cf

Reply:
I believe that the transport port was deprecated a long time ago, and replaced with the HTTP port.